src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static String line = "__________________________________________________________";
+    private static Task[] inputList = new Task[101];
+    private static int numTasks = 0;
+
+    public static void printAddTask(Task t) {
+        //print to show added to list
+        System.out.println(line);
+        System.out.println("Got it. I've added this task: \n" + t);
+        System.out.println("Now you have " + numTasks + " tasks in the list.");
+        System.out.println(line);
+    }
+
+    public static void validTask(String[] userInput) throws DukeException{
+        if (userInput.length < 2 && (userInput[0].equals("todo") ||
```

Comment:
Try not to use magic numbers in your code and give the numbers a more meaningful name